### PR TITLE
Strip Console Statements

### LINKED ISSUE #190: Strip consoles
There aren't that many `console` methods, but it would be nice to have a catch-all method of stripping out any `console` calls that make it through.  This would be via some plugin through webpack.

### PR DESCRIPTION
**Pull request by chrsdietz**
Fixes #190

Changes in this pull request include:
- Adding a strip loader that removes console logs from the code in production

### CODE REVIEW

**Comment by chrsdietz** [comment]
Don't merge yet.  PR is needed for testing.

**Comment by chrsdietz** [comment]
Now it's ready.

File: webpack.config.js [hunk 1]
```diff
@@ -46,6 +47,25 @@ if (node_env === "development") {
...
+// Styling loader for scss and css files.
+moduleLoaders.push({
+  test: /\.(scss|css)$/,
+  loader: ExtractTextPlugin.extract('style', 'typings-for-css-modules?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]!postcss!sass')
+});
+// This loader will strip out all console methods.  We don't care if it's in development.
+if (node_env === "production") {
+  moduleLoaders.push({
+    test: /\.tsx?$/,
+    loader: "strip-loader?strip[]=debug,strip[]=console.log,strip[]=console.info,strip[]=console.error,strip[]=console.debug,strip[]=console.time,strip[]=console.timeEnd"
```
Comment (michaelmyers) [inline webpack.config.js]: Should we leave `console.errors`?  They may be helpful in debugging errors in production.

Comment (chrsdietz) [inline webpack.config.js]: Maybe for now, but I think eventually we'll want all consoles removed.

**Pull request merged**